platform/mellanox/iproute2/Makefile

```diff
@@ -0,0 +1,49 @@
+SHELL = /bin/bash
+.ONESHELL:
+.SHELLFLAGS += -e
+
+MAIN_TARGET = iproute2-mlnx_$(IPROUTE2_MLNX_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+IPROUTE2_DEB = iproute2_$(IPROUTE2_MLNX_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+IPROUTE2_TMP := $(shell mktemp -u -d -t ipr.XXXXXXXXXX)
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Remove any stale files
+	rm -rf iproute2-$(IPROUTE2_MLNX_VERSION)
```

Comment:
What if IPROUTE2_MLNX_VERSION is incremented? I believe this won't remove existing folder. Should we make rm -rf iproute2-*

Reply:
Done